sedbackend/apps/cvs/simulation/storage.py

```diff
@@ -37,6 +37,68 @@
     'minutes': TimeFormat.MINUTES
 })
 
+#TODO: Finish method. No checks on file this time since we won't be getting an uploaded file here, it will already be on the server. 
```

Comment:
Is there something left here that needs to be done?

Reply:
Well yes, but this whole system needs a rework since now we are storing files on the backend directly. Would suggest leaving this for a later time.